core/src/Microsoft.Identity.Core.csproj

```diff
@@ -125,7 +125,17 @@
     <Reference Include="System.Xml" />
     <Compile Include="Platforms\Android\**\*.cs" />
   </ItemGroup>
-  
+  <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid7'">
+    <PackageReference Include="Xamarin.Android.Support.CustomTabs">
+      <Version>25.4.0.2</Version>
+    </PackageReference>
+    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat">
+      <Version>25.4.0.2</Version>
```

Comment:
>25.4.0.2 [](start = 15, length = 8)

is this a consistent version used everywhere? Perhaps we should have a component with this version we include to ensure it gets easily set everywhere?

Reply:
Yes it is. Do you want that included with this PR?

Reply:
Can be done in the next PR. Pls. ensure you create a task for this to ensure it is not forgotten.

---
In reply to: [181953341](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1038#discussion_r181953341) [](ancestors = 181953341)

Reply:
Can be done in the next PR. Pls. ensure you create a task for this to ensure it is not forgotten.

---
In reply to: [182131252](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1038#discussion_r182131252) [](ancestors = 182131252)